### DISCUSSION TITLE
What does `downloads` in `default.config.yml` do?

### DISCUSSION THREAD

**Discussion by Anticom** [geerlingguy/mac-dev-playbook#202] [Q&A] [answered]
I wanted to use Ansible for quite some time now to manage my local machine and with fresh hardware on its way, the time has finally come to look into it.

Being an absolute Ansible beginner I found this repo to be an excellent starting point for my own adventures. However looking at the [`default.config.yml`](https://github.com/geerlingguy/mac-dev-playbook/blob/c5c92c1b8ccf4c431f5c5697091cff0d3dafb7dc/default.config.yml#L2), I just can't figure out, what `downloads` is being used for.

I didn't want to raise an issue since I'm not 100% certain this is a leftover from previous refactorings, but to my untrained eyes it looks exactly like that.

Maybe someone can shed some light on what this actually does (if any)?

**Comment by geerlingguy** [comment, marked as answer]
Indeed, it is an ancient artifact, from 10 years ago now: https://github.com/geerlingguy/mac-dev-playbook/commit/8f8ea4a8e54a6cf65e85196f303d33ce732f6034

Would you like to file a PR to remove that unused var?

**Comment by Anticom** [comment]
@geerlingguy Thanks for the clarification, I was blaming `default.config.yml` but although I stumbled across 8f8ea4a myself, I couldn't figure it out for sure.

For some reason, github didn't want to auto-link the PR, so here it is: [!203](https://github.com/geerlingguy/mac-dev-playbook/pull/203).
